app/views/invoices/_form.html.erb

```diff
@@ -1,102 +1,152 @@
-<%= form_with(model: invoice, url: event_invoices_path(@event), local: true, html: { "x-data" => "{ amount: null }" }) do |form| %>
-  <%= form_errors invoice %>
-  <%= form_errors sponsor %>
-
-  <% if @event.sponsors.count > 0 %>
-    <div class="field">
-      <%= form.label :sponsor, "Sponsor" %>
-      <%= form.select :sponsor,
-        options_for_select(!organizer_signed_in? ? [] : @event.sponsors.select(:id, :name, :contact_email, :address_line1, :address_line2, :address_city, :address_state, :address_postal_code, :address_country).map { |s|
-          [s.name, s.id, { 'data-json': s.to_json }]
-        }),
-        include_blank: "New sponsor…" %>
-    </div>
-  <% end %>
-
-  <%= form.fields_for :sponsor_attributes do |form| %>
-    <h2>Sponsor details</h2>
-
-    <p data-behavior="sponsor_update_warning" style="display: none;" class="flex items-center info line-height-3">
-      <%= inline_icon "important", size: 32, class: "mr1" %>
-      You're currently updating a previous sponsor. This will also update all past invoices to them.
-    </p>
-
-    <%= form.hidden_field :id %>
-    <div class="field mt2">
-      <%= form.label :name, "Name (use company name if applicable)" %>
-      <%= form.text_field :name, placeholder: "Raviga Capital", required: true, disabled: @event.demo_mode? %>
-    </div>
-
-    <div class="field">
-      <%= form.label :contact_email, "Contact email (email to send invoice to)" %>
-      <%= form.email_field :contact_email, placeholder: "peter.gregory@ravigacapital.com", required: true, disabled: @event.demo_mode? %>
-    </div>
-
-    <div class="field">
-      <%= form.label :address_line1, "Street address" %>
-      <%= form.text_field :address_line1, placeholder: "1 Letterman Drive", required: true, disabled: @event.demo_mode? %>
-    </div>
-
-    <div class="field">
-      <%= form.text_field :address_line2, placeholder: "Suite 500", disabled: @event.demo_mode? %>
-    </div>
-
-    <div class="flex">
-      <div class="field flex-auto mr1">
-        <%= form.label :address_city, "City" %>
-        <%= form.text_field :address_city, placeholder: "San Francisco", required: true, disabled: @event.demo_mode? %>
+<% if @event.demo_mode? %>
+  <div class="muted mt-6">
+    Once your organization is activated, you'll be able to create and send invoices
+  </div>
+<% else %>
+  <%= form_with(model: invoice, url: event_invoices_path(@event), local: true, data: { controller: "invoice-form" }, html: { "x-data" => "{ amount: null }" }) do |form| %>
```

Comment:
See if there's a way to run HTML form validations for only the first page when clicking the Continue button.

At the moment, if the 1st page contains an invalid input, and the user goes to the 2nd page and attempts to submit. It doesn't anything. HTML attends to render validation errors for the field, but the field is hidden because it's on the first page.

Reply:
Same things applies to validation error for collapsed sponsor fields.